network/servicenetwork/integr_test.go

```diff
@@ -116,12 +117,52 @@ func (s *testSuite) TestTwoNodesConnect() {
 
 	s.waitForConsensus(2)
 
+	maxNodesToAdd := int(float64(s.getNodesCount()) * claimhandler.NodesToJoinPercent)
```

Comment:
move this calculation to a separate method in `testSuite`, e.g. `GetMaxJoiners`

Reply:
done